app/src/main/java/org/oppia/app/profile/AdminAuthActivityPresenter.kt

```diff
@@ -24,11 +24,14 @@ class AdminAuthActivityPresenter @Inject constructor(
 
   /** Binds ViewModel and sets up text and button listeners. */
   fun handleOnCreate() {
-    activity.title = activity.getString(R.string.add_profile_title)
-    activity.supportActionBar?.setDisplayHomeAsUpEnabled(true)
-    activity.supportActionBar?.setHomeAsUpIndicator(R.drawable.ic_close_white_24dp)
-
     val binding = DataBindingUtil.setContentView<AdminAuthActivityBinding>(activity, R.layout.admin_auth_activity)
+
+    binding.adminAuthToolbar.setNavigationIcon(R.drawable.ic_close_white_24dp)
+    binding.adminAuthToolbar.setNavigationOnClickListener {
+      (activity as AdminAuthActivity).finish()
+    }
+    binding.adminAuthToolbar.title = activity.getString(R.string.add_profile_title)
```

Comment:
Also, this is not needed, xml file already has this attribute

Reply:
Removed.